packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -118,11 +128,12 @@ export const registerScaledPriceAuthority = async (
   { options: { interchainAssetOptions } },
 ) => {
   const {
-    keyword,
-    oracleBrand,
+    keyword: kwd,
+    issuerName = kwd,
+    oracleBrand = issuerName,
```

Comment:
You have a nice and clear progressive override below. Why not use it here?

```suggestion
    keyword,
    issuerName = keyword,
    oracleBrand = issuerName,
```